src/lib/IFrameCommunicatorTypes.ts

```diff
@@ -117,6 +122,8 @@ export enum IFrameMessageType {
   AddInitial = 'kontent-smart-link:add:initial',
   AddAction = 'kontent-smart-link:add:action',
   RefreshPreview = 'kontent-smart-link:preview:refresh',
+  PreviewIFrameCurrentUrl = 'kontent-smart-link:preview:currentUrl',
```

Comment:
Sorry, that I didn't notice it earlier, but `currentUrl` should actually be `current-url` here to be consistent with other event names where we follow this convention.

Reply:
fixed, thanks